.github/workflows/go.yml

```diff
@@ -0,0 +1,43 @@
+name: Push
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - '*'
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
+      run: |
+        go get -v -t -d ./...
+
+    - name: Build
+      run: make all
+
+    - name: Test
+      run: make test
+    
+    - name: Get Tag
+      id: get_tag
+      run: |
+        echo ::set-output name=TAG::$(echo $GITHUB_REF | cut -d / -f 3)
+        
+    - name: Publish container image
+      run: |
+        docker login -u ${DOCKER_USER} -p ${DOCKER_PASSWORD}
```

Comment:
Do you need to explicitly populate the env vars, per [this doc](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#example-using-bash)? It's not totally clear on whether these are automatically populated from the repo's secrets or not.

Reply:
probably not. let me try that out.

Reply:
OK that works. Removed the env vars